microsoft-edge/webview2/concepts/overview-features-apis.md

```diff
@@ -0,0 +1,1231 @@
+---
+title: Overview of WebView2 features and APIs
+description: Overview of WebView2 features and APIs.
+author: MSEdgeTeam
+ms.author: msedgedevrel
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.technology: webview
+ms.date: 06/30/2022
+---
+# Overview of WebView2 features and APIs
+
+By including WebView2 in your app, your app can use various methods and properties provided by the WebView2 control to easily add many kinds of browser features.
```

Comment:
A little confused with the intro to this doc. I would hope for a little more guidance into why I would want to read this doc. In addition why are we only speaking about browser features in this intro, when there are many other things you can do with wv2? Try something like below.

```suggestion
Embedding the WebView2 control in your app gives it access to various methods and properties provided in WebView2 interfaces. We have hundreds of APIs that provide a vast set of capabilities ranging from enhance native capabilities or allow you to modify browser experiences. The goal of this document is to provide a high-level grouping for WebView2 apis to help contextualize the different things you can do with WebView2. 
```

Reply:
@jasonstephen15 - "Guidance on why I would want to read this doc" - Here is previous draft text from top of article, maybe we should add something along these lines, but tighter.

This article provides:
*  A high-level understanding of the capabilities of the WebView2 technology.
*  Concepts about what you can do with the WebView2 APIs.
*  High-level description of WebView2 technology for app developers.
*  Better fundamentals and understanding of WebView2 technology.

Reply:
Embedding the WebView2 control in your app gives it access to various methods and properties provided via the WebView2 interfaces.  WebView2 has hundreds of APIs that provide a vast set of capabilities, ranging from enhancing your app's native-platform capabilities to enabling your app to modify browser experiences.  This article provides a high-level grouping of WebView2 APIs to help you understand the different things you can do using WebView2.